subliminal/providers/addic7ed.py

```diff
@@ -272,8 +273,14 @@ def query(self, series, season, year=None, country=None):
         return subtitles
 
     def list_subtitles(self, video, languages):
-        return [s for s in self.query(video.series, video.season, video.year)
-                if s.language in languages and s.episode == video.episode]
+        titles = [video.title] + video.alternative_series
```

Comment:
Isn't it `video.series` instead of `video.title`?

Reply:
yes

@ratoaq2 can you fix it please?